File: 1894-merge-strings-alternately/1894-merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        string storeAlternative;
        int i = 0,j = 0; 
        while( i<word1.size() && j<word2.size()){
            storeAlternative += word1[i++];
            storeAlternative += word2[j++];
        }
        while( i < word1.size()){ storeAlternative += word1[i++];}
        while( j < word2.size()){ storeAlternative += word2[j++];}
        return storeAlternative;
    }
};

File: 1894-merge-strings-alternately/1894-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        str_com = ""
        loop = min(len(word1), len(word2))
        c = 0
        for c in range(loop):
            str_com+= word1[c]+word2[c]
        
        if len(word1) > len(word2):
            str_com += word1[c+1:]
        elif len(word1) < len(word2):
            str_com += word2[c+1:]

        return str_com